staging/src/k8s.io/client-go/examples/out-of-cluster-client-configuration/main.go

```diff
@@ -62,15 +62,34 @@ func main() {
 		// Examples for error handling:
 		// - Use helper functions like e.g. errors.IsNotFound()
 		// - And/or cast to StatusError and use its properties like e.g. ErrStatus.Message
-		_, err = clientset.CoreV1().Pods("default").Get("example-xxxxx", metav1.GetOptions{})
+
+		// Fill this out if you wish to limit it to a namespace
+		namespace := ""
+
+		pods, err = clientset.CoreV1().Pods(namespace).List(metav1.ListOptions{})
 		if errors.IsNotFound(err) {
 			fmt.Printf("Pod not found\n")
 		} else if statusError, isStatus := err.(*errors.StatusError); isStatus {
 			fmt.Printf("Error getting pod %v\n", statusError.ErrStatus.Message)
 		} else if err != nil {
 			panic(err.Error())
 		} else {
-			fmt.Printf("Found pod\n")
+			fmt.Printf("Found pods\n")
+
+			// Example of retrieving pod names, creationtimestamps by iterating through the Items
+			for _, pod := range pods.Items {
+				fmt.Println("######################")
```

Comment:
Nit: Alternative: add `- ` in front of `Pod name:` and indent containers a bit more. Would be less ugly.